tests/keras_tests/models_tests/test_networks_runner.py

```diff
@@ -103,25 +103,22 @@ def representative_data_gen():
 
         core_config = mct.core.CoreConfig(quantization_config=qc)
         if self.gptq:
-            arc = mct.gptq.GradientPTQConfig(n_iter=2, optimizer=tf.keras.optimizers.Adam(
+            arc = mct.gptq.GradientPTQConfig(n_epochs=1, optimizer=tf.keras.optimizers.Adam(
```

Comment:
maybe set the epochs to 2? I remember having undetected issues in GPTQ that occurred in multiple epochs and we didn't see them because all of our tests were with 1 epoch